src/gui/gui.cpp

```diff
@@ -2414,6 +2425,25 @@ bool FurnaceGUI::loop() {
         }
         ImGui::EndMenu();
       }
+      if (ImGui::BeginMenu("export ZSM...")) {
```

Comment:
Suggestion: hide if no VERA or OPM present?

Reply:
I've added that to the PR

Reply:
Do I need to do anything to add my additional two commits to the PR?

Reply:
Nope - should be automatic.